README.md

```diff
@@ -147,6 +148,43 @@ WASI host must:
 A WASI host that implements the above should be able to spawn threads for a
 variety of languages.
 
+### Thread group
+
+* The main thread starts with a thread group which only contains
+  the main thread.
+
+* Threads created by a thread in a thread group using `wasi_thread_spawn`
+  is added to the thread group.
+
+* When a thread is terminated, it's removed from the thread group.
```

Comment:
Do we even need a concept of thread group?   Why not just say that `wasi_thread_spawn` creates new threads and `wasi_thread_exit` terminates the current thread?   

The thread group concept seems like something fairly specific to linux and/or the world outside of the process. 

If we do want to give a name the container of threads why not use the commonly used term "process"?

Reply:
i just wanted a name for what wasi proc_exit terminates. the name "process" is fine for me too.